### PR TITLE
Sticky videos

### PR DESCRIPTION
**Pull request by manufer20**
Adds a picture-in-picture to the video. The code is optimized for one video per page, but can be changed to fit another requirements.

You can try how it works on docs/videotutorials/try-sticky-video.html

### CODE REVIEW

**Comment by TheWolf534** [comment]
Finalized picture-in-picture prototype for demonstration

**Comment by zerothi** [comment]
Thanks for this @TheWolf534 lets finalize this next week! :)

**Comment by TheWolf534** [comment]
Now sticky videos work on all the videos and also when scrolling upwards

File: docs/conf.py [hunk 1]
```diff
@@ -516,3 +516,8 @@ def setup(app):
         app.connect("include-read", rstjinja_include)
     except BaseException as e:
         print("cannot do jinja-replacements on included files")
+
+
+def setup(app):
+    app.add_js_file("sticky_video.js")
```
Comment (zerothi) [inline docs/conf.py:522]: there is already an `setup` method, so I don't think this will do anything. At least I don't think it should be necessary.

**Review by zerothi — CHANGES_REQUESTED** [review]
small changes, and a rebase would be required.

**Pull request closed without merging**